### DISCUSSION TITLE
May I contribute on your project?

### DISCUSSION THREAD

**Discussion by imranertaza** [edrisranjbar/qurandl#17] [General]
Hi
I am interested to contribute to your project. Just to be sure, may I contribute to your project, please?
If yes, I think, I should fork the develop branch to make changes. Am I right?

**Comment by edrisranjbar** [comment]
Hi yes you are more than welcome to participate in this open source project an actually we'll be happy to have you here in this none-profit project. we didn't provide a specific code of conduct for this project **yet**; but in order to contribute you can do the following steps.

1. Fork the repo into you github account
2. Create a new branch and work on that
3. Request a new pull request

you can use[ this docs](https://github.com/firstcontributions/first-contributions)

Thanks 🌹

**Reply by imranertaza** [reply]
Thank you for you time. I hope I will start contributing on this project soon. I like this project. I have seen that your develop branch is more than 24 commit ahead of the main branch. Means, you have develop branch which is updated. right? So, I should checkout on the develop branch and start working on it. Am I correct?

**Comment by edrisranjbar** [comment]
yes you're right; you must start from development branch's last commit.

**Reply by imranertaza** [reply]
Ok. Thank you. I think it would be better for the contributor if you set the develop branch as the default branch. Then, contributors will be able to fork the develop branch as default easily.

**Comment by edrisranjbar** [comment]
great suggestion, I did that and now; `develop` is the default branch,
